web/themes/custom/votegov/templates/node/node--state-territory.html.twig

```diff
@@ -125,46 +126,61 @@
-          <hr>
-          {% if nvrf_details.heading %}
-            <h3>{{ nvrf_details.heading }}</h3>
-          {% endif %}
-
-          {{ nvrf_details.text }}
-
-          {% if nvrf_details.link_text %}
-            {# Set path to form dynamically using page route. #}
-            {% set form_route = language != 'es' ? 'vote_nvrf.nvrf_page' : 'vote_nvrf.nvrf_page_es' %}
-            {% set state_name = currentnode.label | lower | replace({' ': '-'}) %}
-
-            {% include '@votegov/component/button.html.twig' with {
-              'href': 'internal:' ~ path(form_route, {'state_name': state_name}),
-              'label': nvrf_details.link_text,
-              'data_label': 'data-state-ext',
-              'data': 'nvrf-pdf-tool-registration'
-            } %}
-          {% endif %}
-        {% endset %}
+
+    {% if has_mail and (mail_registration is not empty) %}
+  {% set mail_body %}
+    <p><strong>{{ 'Mail-in registration deadline:' | t }}</strong> {{ bymail_deadline }}</p>
+    {{ mail_registration.text }}
+
+    {% if state_mail_pdf_link and mail_registration.link_text %}
+      <p>{{ link(mail_registration.link_text, state_mail_pdf_link) }}</p>
+    {% endif %}
+  {% endset %}
+
+  {% set mail_footer %}
+    {% if accepts_nvrf and nvrf_details is not empty %}
+      <hr>
+      {% if nvrf_details.heading %}
+        <h3>{{ nvrf_details.heading }}</h3>
       {% endif %}
 
-      {% include '@votegov/component/info-card.html.twig' with {
-        'heading': mail_registration.heading,
-        'body': mail_body | render | trim | t({'@state_name': state_name}),
-        'footer': mail_footer
-      } %}
+      {{ nvrf_details.text }}
+
+      {% if nvrf_details.link_text %}
+        {# Set path to form dynamically using page route. #}
+        {% set form_route = language!= 'es'? 'vote_nvrf.nvrf_page' : 'vote_nvrf.nvrf_page_es' %}
+        {% set state_name = currentnode.label | lower | replace({' ': '-'}) %}
+
+        {% include '@votegov/component/button.html.twig' with {
+          'href': 'internal:' ~ path(form_route, {'state_name': state_name}),
+          'label': nvrf_details.link_text,
+          'data_label': 'data-state-ext',
+          'data': 'nvrf-pdf-tool-registration'
+        } %}
+      {% endif %}
     {% endif %}
+  {% endset %}
+
+  {% include '@votegov/component/info-card.html.twig' with {
+    'heading': mail_registration.heading,
+    'body': mail_body | render | trim | t({'@state_name': state_name}),
+    'footer': mail_footer
+  } %}
+{% else %}
+  {% if not has_mail and (no_mail_registration is not empty) %}
+    {% set body = no_mail_registration.text['#markup'] | t({'@state_name': title_english}) %}
+    {% set link_title = no_mail_registration.link_text['#markup'] | t({'@state_name': title_english }) %}
+
+    {% include '@votegov/component/info-card.html.twig' with {
+      'heading': no_mail_registration.heading,
+      'body': body,
+      'link': {
+        'url': registration_link,
```

Comment:
Change this backup link to the "more info" link that is in the in-person block. Since that one is the general election link, instead of an online reg link that may not be set.

Reply:
good catch.. update made